examples/qcodesrc_example.json

```diff
@@ -0,0 +1,15 @@
+{
+    "core":{
+        "loglevel": "DEBUG",
+        "default_fmt": "data/{date}/#{counter}_{name}_{time}"
```

Comment:
Will this not create a folder ` C:\Users\Jens\src\data\data\...` i.e. with data twice?